tests/functional/codegen/calling_convention/test_default_function.py

```diff
@@ -25,14 +28,15 @@ def test_basic_default(w3, get_logs, get_contract_with_gas_estimation):
 def __default__():
     log Sent(msg.sender)
     """
-    c = get_contract_with_gas_estimation(code)
-
-    logs = get_logs(w3.eth.send_transaction({"to": c.address, "value": 10**17}), c, "Sent")
-    assert w3.eth.accounts[0] == logs[0].args.sender
-    assert w3.eth.get_balance(c.address) == w3.to_wei(0.1, "ether")
+    c = get_contract(code)
+    env.set_balance(env.deployer, 10**18)
+    env.message_call(c.address, value=10**17)  # call default function
```

Comment:
for clarity, we should add `data=b""` as a kwarg to `message_call()` invocations in this file